nservicebus/upgrades/5to6-endpoint-name-helper.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Helper class for Derived endpoint name behavior in Version 6
+summary: Describes how to derive an endpoint name
+---
+
+This algorithm is used in Version 5 of NServiceBus to determine Endpoint Name if none is provided.
```

Comment:
Fair enough. This doesn't appear in the menu and the only link to it contains a description of why. I guess people could still come here via search or direct link though.